### PR TITLE
Fix local KTX files never being removed from pending downloads

### PR DESCRIPTION
**Pull request by jherico**
Fix for https://highfidelity.manuscript.com/f/cases/13056

## Testing

In current master builds, the textures for the default avatar will never be removed from the extended stats display and the 'content downloading' bar will never disappear.  With this PR build, this should no longer be the case.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7943/app_12615_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7943/HighFidelity-Beta-PR12615-749f870b47fc22397b725cc9e3a03a88cd8f3d6c.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7949/app_12615_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7949/HighFidelity-Beta-PR12615-7d267bb709a34d1159e4a2509734bef5d4dcc892.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7949/HighFidelity-Beta-PR12615-7d267bb709a34d1159e4a2509734bef5d4dcc892.exe

File: libraries/model-networking/src/model-networking/TextureCache.cpp [hunk 1]
```diff
@@ -468,6 +472,9 @@ void NetworkTexture::makeLocalRequest() {
         path = ":" + _url.path();
     }
 
+    auto self = _self;
```
Comment (ZappoMan) [inline libraries/model-networking/src/model-networking/TextureCache.cpp]: where are you using this local variable `self` I don't see that??

Comment (jherico) [reply libraries/model-networking/src/model-networking/TextureCache.cpp]: It's a holdover from when I was using a lambda in the connection.  Fixed.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7963/app_12615_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7963/HighFidelity-Beta-PR12615-e33539f5752791eaab00784b1aa6e6f67ab9bbbd.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7963/HighFidelity-Beta-PR12615-e33539f5752791eaab00784b1aa6e6f67ab9bbbd.exe

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by qlimnavarro** [comment]
Pass @conklin94122 
Jumped to various domains with and without HMD and no downloads were left in pending.

**Comment by conklin94122** [comment]
@jherico Needs dev review

**Pull request merged**